src/AbstractWalkers/helpers.jl

```diff
@@ -0,0 +1,129 @@
+# show methods
+
+"""
+    custom_sort(arr::Vector{Int}, period::Int)
+
+Sorts [1,2,3,4,5,6,7,8] into [1,3,5,7,2,4,6,8] for period = 2. Useful for printing triangular lattices.
+
+"""
+function custom_sort(arr::Vector{Int}, period::Int)
```

Comment:
There might be a built-in one, I haven't fully explored it. I wrote this one and didn't check what's available online. Since it's just a `show` method I didn't put too much effort into it. Will have another look later.